diff --git a/barretenberg_static_lib/Cargo.toml b/barretenberg_static_lib/Cargo.toml
index 7598bae..6315fc1 100644
--- a/barretenberg_static_lib/Cargo.toml
+++ b/barretenberg_static_lib/Cargo.toml
@@ -8,14 +8,15 @@ edition = "2021"
 [dependencies]
 
 common = { path = "../common" }
-barretenberg_wrapper = { git = "https://github.com/noir-lang/aztec-connect", branch = "kw/noir-dsl" }
+barretenberg_wrapper = { path = "../../aztec-connect/barretenberg_wrapper" }
+# barretenberg_wrapper = { git = "https://github.com/noir-lang/aztec-connect", branch = "kw/noir-dsl" }
 
 sha2 = "0.9.3"
 blake2 = "0.9.1"
 sled = "0.34.6"
 
 dirs = "3.0"
-downloader = { version = "0.2.6" }
+downloader = { version = "= 0.2.6" }
 tempfile = "3.3.0"
 indicatif = "0.15.0"
 regex = "1.4.0"
