diff --git a/crates/nargo/Cargo.toml b/crates/nargo/Cargo.toml
index cb953e36..ef4c867e 100644
--- a/crates/nargo/Cargo.toml
+++ b/crates/nargo/Cargo.toml
@@ -29,7 +29,8 @@ hex.workspace = true
 tempdir.workspace = true
 
 # Backends
-aztec_backend = { optional = true, package = "barretenberg_static_lib", git = "https://github.com/noir-lang/aztec_backend", rev = "8c8ec2dd9c376bef598d17f023fac172e5e47860" }
+aztec_backend = { optional = true, path = "../../../aztec_backend/barretenberg_static_lib"}
+# aztec_backend = { optional = true, package = "barretenberg_static_lib", git = "https://github.com/noir-lang/aztec_backend", rev = "8c8ec2dd9c376bef598d17f023fac172e5e47860" }
 marlin_arkworks_backend = { optional = true, git = "https://github.com/noir-lang/marlin_arkworks_backend", rev = "601e24dcb5dcbe72e3de7a33879aaf84e171d541" }
 
 [features]
diff --git a/crates/nargo/build.rs b/crates/nargo/build.rs
index 6441ffda..6ed1b80c 100644
--- a/crates/nargo/build.rs
+++ b/crates/nargo/build.rs
@@ -63,8 +63,4 @@ pub fn copy<U: AsRef<Path>, V: AsRef<Path>>(from: U, to: V) -> Result<(), std::i
 }
 
 fn main() {
-    let stdlib_src_dir = Path::new("../../noir_stdlib/");
-    rerun_if_stdlib_changes(stdlib_src_dir);
-    let target = dirs::config_dir().unwrap().join("noir-lang").join("std");
-    copy(stdlib_src_dir, target).unwrap();
 }
